linode/manager
# M3-5934: Fix MUI v5 TypeScript errors

## Description
Fix errors from TypeScript caused by the MUI v5 upgrade.

- [x] theme.spacing() errors
- [x] Newly-removed props and other prop adjustments
- [x] theme.breakpoints.down/up() arguments
- [x] Typing issues

## How to test
Confirm that the app compiles and works, and check the VSCode shell and the Dev Tools console to see if any errors are logged.
